src/mscorlib/src/System/Diagnostics/Stacktrace.cs

```diff
@@ -664,6 +669,35 @@ private static bool ShowInStackTrace(MethodBase mb)
             return !(mb.IsDefined(typeof(StackTraceHiddenAttribute)) || (mb.DeclaringType?.IsDefined(typeof(StackTraceHiddenAttribute)) ?? false));
         }
 
+        private static Type ResolveStateMachineMethod(ref MethodBase mb, Type stateMachineType)
```

Comment:
Why is `mb` passed by ref but `stateMachineType` passed as an arg and then returned?

Reply:
Change to both be ref